### ISSUE TITLE
mediaWatch polyfill implementation needed for IE9< 

### ISSUE THREAD

**Issue by anselmbradford** [codeforamerica/ohana-web-search#227]
Enquire-js is used to disable the about popups on mobile screen sizes. Currently it is disabled because it was throwing errors in IE9< (https://github.com/codeforamerica/ohana-web-search/blob/master/app/assets/javascripts/app/popup-manager.js#L19). See http://stackoverflow.com/questions/14402932/enquire-js-legacy-support. This should be refactored to handle the popups properly on mobile screen sizes.
